flybywire-aircraft-a320-neo/en-US.locPak

```diff
@@ -19,6 +19,8 @@
       "A32NX.LOADING.TIPS.A320NEO_004": "There are 9693 A320 family aircraft built! That's almost as much as the population of Tuvalu!",
       "A32NX.LOADING.TIPS.A320NEO_005": "Airbus A320neo aircraft have a range of 3500 NM or 6500 KM. That's enough to fly from New York to Paris!",
       "A32NX.LOADING.TIPS.A320NEO_006": "Did you know? The A320 family has 4 variants! The A318, A319, A320, and A321."
```

Comment:
Missing a `,`on the end of this line.

Reply:
Thanks, now added. Probably removed it by accident